integration_tests/test_env_populator.py

```diff
@@ -12,7 +12,7 @@
 #create robyn.env before test and delete it after test
 @pytest.fixture
 def env_file():
-    CONTENT = "PORT=8080"
+    CONTENT = "ROBYN_PORT=8080" + "\n" + "ROBYN_URL=127.0.1.1"
```

Comment:
You can use python multiline strings(https://www.w3schools.com/python/gloss_python_multi_line_strings.asp) here.

Reply:
done